sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/samples/simple-chat/js/functionapp/addToGroup/index.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+module.exports = function (context, req) {
```

Comment:
@weshaggard similar question around having JS samples.

Reply:
Similar answer they should live next to the JS library in the JS repo.